src/Microsoft.Health.Dicom.Core/Extensions/JsonSerializerOptionsExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using EnsureThat;
+using FellowOakDicom.Serialization;
+using Microsoft.Health.Dicom.Core.Serialization;
+
+namespace Microsoft.Health.Dicom.Core.Extensions
+{
+    /// <summary>
+    /// Provides <see langword="static"/> methods configuring <see cref="JsonSerializerOptions"/>.
+    /// </summary>
+    public static class JsonSerializerOptionsExtensions
+    {
+        /// <summary>
+        /// Configures an instance of <see cref="JsonSerializerOptions"/> for usage within
+        /// the DICOM server and related services.
+        /// </summary>
+        /// <param name="options">A set of existing options.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="options"/> is <see langword="null"/>.</exception>
+        public static void ConfigureDefaultDicomSettings(this JsonSerializerOptions options)
```

Comment:
General note - why specify default settings here? Do we want to pin to this version of defaults?

Reply:
Well, it is really in case of breaking changes in the underlying json serializer libs. If those defaults change in like v7, our web API should change its behavior (unless we explicitly want it to)

Reply:
As agreed offline, we'll keep these defaults